galleon-pack/src/main/resources/configs/standalone/standalone-microprofile.xml/config.xml

```diff
@@ -7,12 +7,14 @@
         <include name="microprofile-fault-tolerance"/>
         <include name="microprofile-jwt"/>
         <include name="microprofile-openapi"/>
+        <include name="microprofile-opentracing"/>
         <include name="undertow-legacy-https"/>
         <exclude name="management-security-realm"/>
     </layers>
     <!-- TODO WFLY-13150 encapsulate these in a layer -->
     <feature-group name="distributable-web"/>
     <feature-group name="infinispan-local-web"/>
+    <feature-group name="microprofile-opentracing-jaeger"/>
```

Comment:
The current standalone-microprofile-ha.xml uses the empty config.

Reply:
ok, I'e realigned the produced configurations with the ones in 21